js/fieldmanager-loader.js

```diff
@@ -4,22 +4,71 @@
  * Ensures that metaboxes have loaded before initializing functionality.
  * @param {function} callback - The callback function to execute when the DOM is ready.
  */
-function fmLoadModule( callback ) {
+ function fmLoadModule( callback ) {
+	/**
+	 * Wraps the provided callback to add check for the Gutenberg editor. If this
+	 * is called within the Gutenberg editor, then the callback function will trigger
+	 * after the metaboxes are initialized.
+	 */
+	const wrappedCallback = () => {
+		if (document.querySelector('.block-editor-page')) {
+			let pollAttempts = 0;
```

Comment:
Rather than setting this to `0` here and hardcoding the number of attempts it can't be greater than below (`5`) why not set this to `5` (and maybe rename it to something like `pollAttemptsRemaining`) and decrement it instead? Then the check could just be whether there are any poll attempts remaining. Currently, the threshold for polling attempts is pretty far down into the logic.

Reply:
Good suggestion. Updated to decrement the counter in 1f4697f.